config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-periodic-dra.yaml

```diff
@@ -405,3 +405,98 @@ periodics:
               value: "true"
             - name: NODE_PRELOAD_IMAGES
               value: "gcr.io/k8s-staging-perf-tests/sleep:v0.0.3"
+
+  - name: ci-kubernetes-e2e-gce-5000-node-dra-with-workload
+    cluster: k8s-infra-prow-build
+    tags:
+      - "perfDashPrefix: gce-dra-5000Nodes-with-workload"
+      - "perfDashBuildsCount: 270"
+      - "perfDashJobType: performance"
+    # TODO (alaypatel07): increase this interval once stable
+    interval: 12h
```

Comment:
we almost certainly need to use a cron to avoid scheduling conflicts, and also not increase the interval, these are really expensive, and it requires some time to coordinate permission for quota this large in a project without a contract 🙃